docs/source/contributing.rst

```diff
@@ -167,6 +167,14 @@ to get feedback on, feel free to submit it as a PR. (In this case it's
 traditional to start the PR title with ``[WIP]``, for "work in
 progress".)
 
+When you are submitting your PR, you can include ``Closes #123``,
+``Fixes: #123`` or
+`some variation <https://help.github.com/en/articles/closing-issues-using-keywords>`__
+in either your commit message or the PR description, in order to
```

Comment:
Does this definitely work in commit messages too? If you haven't tested it there, let's just say "in the PR description", as that's the only place I have experience using it.

Reply:
> When a pull request or commit references a keyword and issue number, it creates an association between the pull request and the issue.

From the [documentation](https://help.github.com/en/articles/closing-issues-using-keywords), let me know if you still want to change it!

Reply:
ah great, I haven't actually tried it in a commit message so I trust the docs!